### PR TITLE
irb: add page

### PR DESCRIPTION
**Pull request by tit**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by tit** [comment]
> pages/common/irb.md:3: TLDR004 Command descriptions should end in a period
> Please fix the error(s) and push again.

done!

File: pages/common/irb.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# irb
+
+> Interactive Ruby Shell.
+
+- Run Ruby Sell with load some gem:
```
Comment (sbrl) [inline pages/common/irb.md]: `with load some gem` doesn't appear to read very well - I'm not sure what you're getting at here?

**Comment by sbrl** [comment]
Thanks, @tit! I've left a comment for you to review.

File: pages/common/irb.md (hunk 1 above)
Comment (agnivade) [inline pages/common/irb.md]: Please start with the simplest invocation of the command, in this case `irb`. Keep the description as `Launch a REPL(Read-Eval-Print-Loop) shell`

Comment (agnivade) [reply pages/common/irb.md]: This should be `Launch the shell and load the "net/http" library`

File: pages/common/irb.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+`irb -r {{net/http}}`
+
+- Don't show result:
```
Comment (agnivade) [inline pages/common/irb.md]: What is the use of this option ? Is this used commonly enough ? I don't see a usecase where you launch a REPL shell but do not want to see command output.

**Comment by agnivade** [comment]
Please also sign the CLA.

**Pull request closed without merging**